jsonenums.go

```diff
@@ -75,6 +75,8 @@ import (
 	"path/filepath"
 	"strings"
 
+	"./utils"
```

Comment:
jsonenums.go:78:2: cannot find package "utils" in any of:
	/usr/local/Cellar/go/1.6.2/libexec/src/utils (from $GOROOT)
	/Users/benjamintrapani/Documents/Clypd/gillnet/go/src/utils (from $GOPATH)